util/topgen/lib.py

```diff
@@ -160,7 +165,7 @@ def get_pad_list(padstr):
     width = first - last + 1
 
     for p in range(first, last + 1):
-        pads.append({"name": pad, "index": p})
+        pads.append(OrderedDict({"name": pad, "index": p}))
```

Comment:
@imphil This still cause the generated hjson has `index` field first and `name` later... really weird to me..

Reply:
You create a normal dict with `{` in your ordered dict -- so by the time your data/value pairs reach the `OrderedDict` constructor, they are already unordered. 

Use this instead for a Python 3.5-compatible solution:

```suggestion
        pads.append(OrderedDict([("name", pad), ("index", p)]))
```

The same is true for all the other cases where you construct an `OrderedDict`.